CHANGELOG.adoc

```diff
@@ -12,6 +12,7 @@ The following classes have been renamed to reflect their new usage:
 * Rename `UseCaseUsageActorsCompartmentNodeDescriptionProvider` to `CaseUsageActorsCompartmentNodeDescriptionProvider`
 * Rename `UseCaseUsageObjectiveRequirementCompartmentNodeDescriptionProvider` to `CaseUsageObjectiveRequirementCompartmentNodeDescriptionProvider`
 * Rename `UseCaseUsageSubjectCompartmentNodeDescriptionProvider` to `CaseUsageSubjectCompartmentNodeDescriptionProvider`
+* Repaced `AstTreeParser.resolveAllProxy` and `AstTreeParser.resolveAllImport` by `AstTreeParser.resolveAllReference`
```

Comment:
Repaced -> Replaced
Furthermore, that's not accurate enough. Please list ALL deleted APIs.